src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ParameterNameCheck.java

```diff
@@ -72,6 +78,12 @@
      */
     private boolean ignoreOverridden;
 
+    /** The visibility scope where methods are checked. */
+    private Scope scope = Scope.ANONINNER;
```

Comment:
why you selected this value as default ?
I think it should be NOTHING, or null or absent of Optional.